Bio/PDB/parse_pdb_header.py

```diff
@@ -21,6 +20,23 @@
 from Bio import File
 
 
+# Added to parse SSBOND records from the PDB header and add them back to the header dict
+# Eric G. Suchanek, PhD 11/25/22
+def _get_ssbond(inl):
+    # SSBOND   1 CYS A   26    CYS A   84                          1555   1555  2.04
+    # CYS A   26    CYS A   84                          1555   1555  2.0
+    # print(inl)
+    tok = inl.split()
```

Comment:
Please do not use split when dealing with PDB files. The format is character-delimited, so you might run into cases where e.g. the residue number is large enough to leave no space between itself and the next field and split will fail.

See [here](https://www.wwpdb.org/documentation/file-format-content/format33/sect6.html#SSBOND) for the description of the SSBOND records.